Docs/Configuration.tex

```diff
@@ -265,55 +265,64 @@ \subsection{Configuration Terms}\label{configuration-terms}
   \texttt{invalid\ value} if missing.
 \item
   \texttt{fatal\ behaviour} --- behaviour leading to boot termination.
-  Implementation must stop the boot process from going any further until
-  next host system boot. It is allowed but not required to perform cold
-  reboot or show any warning message.
+  Implementations shall prevent the boot process from continuing until
+  the next host system boot. It is permitted, but not required, to
+  execute cold reboots or to show warning messages in such cases.
 \item
   \texttt{undefined\ behaviour} --- behaviour not prescribed by this
-  document. Implementation is allowed to take any measures including but
-  not limited to \texttt{fatal\ behaviour}, assuming any states or
-  values, or ignoring, unless these measures negatively affect system
-  security in general.
+  document. Implementations may take any measures including, but not
+  limited to, measures associated with \texttt{fatal\ behaviour}, assumptions
+  of any state or value, or disregarding any associated states or values. This is
+  however subject to such measures not negatively impacting upon system integrity.
 \end{itemize}
 
 \subsection{Configuration Processing}\label{configuration-processing}
 
-\texttt{OC\ config} is guaranteed to be processed at least once if it
-was found. Depending on OpenCore bootstrapping mechanism multiple
-\texttt{OC\ config} files may lead to reading any of them. No
-\texttt{OC\ Config} may be present on disk, in which case all the values
-read follow the rules of \texttt{invalid\ value} and
-\texttt{optional\ value}.
+The \texttt{OC\ config} file is guaranteed to be processed at least once if found.
+Depending on OpenCore bootstrapping mechanism, the presence of multiple
+\texttt{OC\ config} files may lead to the reading of any of them. It is permissible
+for no \texttt{OC\ Config} file to be present on disk. In such cases, all values
+shall follow the rules of \texttt{invalid\ values} and \texttt{optional\ values}.
 
-\texttt{OC\ config} has size, nesting, and key amount limitations.
-\texttt{OC\ config} size does not exceed \texttt{16\ MBs}.
-\texttt{OC\ config} has no more than \texttt{8} nesting levels.
-\texttt{OC\ config} has up to \texttt{16384} XML nodes (i.e.~one
-\texttt{plist\ dictionary} item is counted as a pair of nodes) within
-each \texttt{plist\ object}.
+The \texttt{OC\ config} file has restrictions on size, nesting levels and number of keys:
 
-Reading malformed \texttt{OC\ config} file leads to
-\texttt{undefined\ behaviour}. Examples of malformed \texttt{OC\ config}
-cover at least the following cases:
+\begin{itemize}
+\tightlist
+\item
+  The \texttt{OC\ config} file size shall not exceed \texttt{16\ MBs}.
+\item
+  The \texttt{OC\ config} file shall not have more than \texttt{8} nesting levels.
+\item
+  The \texttt{OC\ config} file may have up to \texttt{16384} XML nodes within
```

Comment:
Actually this is outdated. Maximum size is now 32 MB, maximum nesting levels is 32, maximum node count is 32768.

Reply:
Will update accordingly